helix-core/src/main/java/org/apache/helix/task/TaskDriver.java

```diff
@@ -526,6 +530,163 @@ public void enqueueJobs(final String queue, final List<String> jobs,
     }
   }
 
+  /**
+   * Add task to a running (IN-PROGRESS) job or the job which is not started yet. Timeout for this
+   * operation is default timeout
```

Comment:
"default timeout" is not clear enough.
Please mention the real timeout here. Maybe use java doc feature to link with the variable would be a good idea. But I'm not sure if it works fine. please have a try.

Reply:
Done.